app/vep/vep_resources.py

```diff
@@ -140,3 +141,30 @@ async def download_results(request: Request, submission_id: str):
     except Exception as e:
         logging.debug(e)
         return response_error_handler(result={"status": 500})
+
+@router.get("/submissions/{submission_id}/results", name="view_results")
+async def fetch_results(request: Request, submission_id: str, page: int, page_size: int):
+    try:
+        workflow_status = await get_workflow_status(submission_id)
+        submission_status = PipelineStatus(submission_id=submission_id, status=workflow_status['workflow']['status'])
+        if submission_status.status == "SUCCEEDED":
```

Comment:
It would be good if we made a enum for status values rather than hardcoded strings. You get auto complete, if the string changes you only have to make one change, reduces the chance of typos, and it makes it explicit to future developers what statuses the service may have to deal with.